cloudflare/resource_cloudflare_page_rule.go

```diff
@@ -578,6 +579,10 @@ func transformFromCloudflarePageRuleAction(pageRuleAction *cloudflare.PageRuleAc
 		value = pageRuleAction.Value.(string)
 		break
 
+	case contains(pageRuleAPIFloatAsStringFields, pageRuleAction.ID):
```

Comment:
Nit: `browser_cache_ttl` is a pretty unique case (similar to `forwarding_url` below). We could probably just do a `pageRuleAction.ID == "browser_cache_ttl"` check here instead of the `contains`. I wouldn't expect to add to `pageRuleAPIFloatAsStringFields`.

```suggestion
	case pageRuleAction.ID == "browser_cache_ttl":
```

Reply:
Similarly, if we do that, we can also do away with `pageRuleAPIFloatFields` only containing `edge_cache_ttl` and put that in a single `pageRuleAction.ID == "edge_cache_ttl"` check.

Reply:
Here's a `diff` of what I mean.

```diff
diff --git a/cloudflare/resource_cloudflare_page_rule.go b/cloudflare/resource_cloudflare_page_rule.go
index 7fcfc3b..4a870c3 100644
--- a/cloudflare/resource_cloudflare_page_rule.go
+++ b/cloudflare/resource_cloudflare_page_rule.go
@@ -540,12 +540,6 @@ var pageRuleAPINilFields = []string{
        "disable_railgun",
        "disable_security",
 }
-var pageRuleAPIFloatFields = []string{
-       "edge_cache_ttl",
-}
-var pageRuleAPIFloatAsStringFields = []string{
-       "browser_cache_ttl",
-}
 var pageRuleAPIStringFields = []string{
        "bypass_cache_on_cookie",
        "cache_key",
@@ -571,15 +565,15 @@ func transformFromCloudflarePageRuleAction(pageRuleAction *cloudflare.PageRuleAc
                value = true
                break

-       case contains(pageRuleAPIFloatFields, pageRuleAction.ID):
-               value = pageRuleAction.Value.(float64) // we use TypeInt but terraform seems to do the right thing converting from float
-               break
-
        case contains(pageRuleAPIStringFields, pageRuleAction.ID):
                value = pageRuleAction.Value.(string)
                break

-       case contains(pageRuleAPIFloatAsStringFields, pageRuleAction.ID):
+       case case pageRuleAction.ID == "edge_cache_ttl"::
+               value = pageRuleAction.Value.(float64) // we use TypeInt but terraform seems to do the right thing converting from float
+               break
+
+       case pageRuleAction.ID == "browser_cache_ttl":
                value = fmt.Sprintf("%.0f", pageRuleAction.Value.(float64))
                break
```